### PR TITLE
[TIMOB-23645] Android: Added Color to Android notification color behind icon

### PR DESCRIPTION
**Pull request by dieskim**
**JIRA**: https://jira.appcelerator.org/browse/TIMOB-23645

- Added Color to Android notification color behind icon
- Added Documentation

### CODE REVIEW

File: android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java [hunk 1]
```diff
@@ -188,13 +188,15 @@ public void setLargeIcon(Object icon)
...
+    @Kroll.method @Kroll.setProperty
+    public void setColor(String color)
+    {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP)) {
+            notificationBuilder.setColor(TiColorHelper.parseColor(color));
+        }
+        setProperty(TiC.PROPERTY_COLOR, color);
+    }
+    
 	@Kroll.method @Kroll.setProperty
```
Comment (hansemannn) [inline android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java]: Please check the indentation. I fixed it previously but you want to check this as well - maybe for future PR's 😙.

**Review by hansemannn — APPROVED** [review]
CR **passed**. Awaiting another peer-review from @ypbnv.

File: android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java [hunk 2]
```diff
@@ -71,6 +72,9 @@ public void handleCreationDict(KrollDict d)
 		if (d.containsKey(TiC.PROPERTY_LARGE_ICON)) {
 			setLargeIcon(d.get(TiC.PROPERTY_LARGE_ICON));
 		}
+        	if (d.containsKey(TiC.PROPERTY_COLOR)) {
```
Comment (ypbnv) [inline android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java]: Fix indentation.

File: android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java [hunk 3]
```diff
@@ -183,7 +187,16 @@ public void setLargeIcon(Object icon)
 		}
 		setProperty(TiC.PROPERTY_LARGE_ICON, icon);
 	}
-
+
```
Comment (ypbnv) [inline android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java]: Remove spaces from empty lines - here and on line 199.

File: android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java [hunk 4]
```diff
@@ -183,7 +187,16 @@ public void setLargeIcon(Object icon)
 		}
 		setProperty(TiC.PROPERTY_LARGE_ICON, icon);
 	}
-
+    
+	@Kroll.method @Kroll.setProperty
+	public void setColor(String color)
+	{
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP)) {
```
Comment (ypbnv) [inline android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java]: Please, import `android.os.Build`
One extra bracket is added after the `if` statement.

**Review by ypbnv — CHANGES_REQUESTED** [review]
Check comments.

File: android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java (hunk 2 above)
Comment (dieskim) [reply android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java]: Done

File: android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java (hunk 3 above)
Comment (dieskim) [reply android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java]: Done

File: android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java (hunk 4 above)
Comment (dieskim) [reply android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java]: Done

**Review by ypbnv — APPROVED** [review]
CR: Pass

File: apidoc/Titanium/Android/Notification.yml [hunk 5]
```diff
@@ -308,8 +308,18 @@ properties:
...
+ 
+  - name: color
+    summary: Accent color used behind icon.
+    description: |
+        Accent color of the circle used behind icon. The Icon will be stenciled in white on top of a 
+        cicle of the color set. For information about color values, see the "Colors" section of <Titanium.UI>.
+        
+        See Android Reference for the [color](https://developer.android.com/reference/android/app/Notification.html#color) property.
+    type: String
+    since: "7.0.0"
```
Comment (garymathews) [inline apidoc/Titanium/Android/Notification.yml]: Also add `osver: {android: {min: "5.0"}}`

**Review by garymathews — CHANGES_REQUESTED** [review]

File: apidoc/Titanium/Android/Notification.yml (hunk 5 above)
Comment (dieskim) [reply apidoc/Titanium/Android/Notification.yml]: Done

**Comment by dieskim** [comment]
sure but I am not sure what to do exactly...I thought it was already linked?

File: apidoc/Titanium/Android/Notification.yml (hunk 5 above)
Comment (dieskim) [reply apidoc/Titanium/Android/Notification.yml]: @garymathews - I see this has not been approved yet?

**Comment by lokeshchdhry** [comment]
FR Passed.

Notification shows the specified `color`.

Studio Ver:     4.10.0.201709271713
SDK Ver:        7.0.0 local build
OS Ver:         10.12.3
Xcode Ver:      Xcode 8.3.3
Appc NPM:       4.2.10
Appc CLI:       6.3.0
Ti CLI Ver:     5.0.14
Alloy Ver:      1.10.7
Node Ver:       7.10.1
Java Ver:       1.8.0_101
Devices:        ⇨ google Nexus 6P --- Android 8.0.0
                ⇨ google Nexus 5 --- Android 6.0.1

**Comment by lokeshchdhry** [comment]
@sgtcoolguy , can you please merge this PR. Its a community PR, probably unit tests can't be added.

**Pull request merged**